.travis/docker-compose-travis.yml

```diff
@@ -2,7 +2,7 @@ version: "2"
 
 services:
   analytics_api:
-    image: edxops/analytics_api:latest
+    image: ednxops/analytics_api:py3
```

Comment:
is the n in ednxops intentional?

Reply:
ah that's for your repo :) i see.

Reply:
yes, that is our hubspot account. Sometimes we use `ednx_` as a prefix for edunext owned repos and in this case it turned out to be confusingly similar to edx's edxops account.

Reply:
However, this PR is never to be merged. It is only here to make it easy for us to find the results of running travis